frontend-public/.babelrc

```diff
@@ -1,12 +1,13 @@
 {
-  "presets": ["env", "react", "stage-2"],
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
   "plugins": [
     [
-      "transform-runtime",
+      "@babel/transform-runtime",
```

Comment:
Is the `@babel/` prefix just added here for verbosity? If memory serves, it's assumed and not needed

Reply:
I think this is an artifact of change with babel v7. The documentation still says that only presets prefixed with `babel-preset-` can be shorthanded. No mentions of `@babel/preset-`